torchgeo/datasets/ucmerced.py

```diff
@@ -143,6 +144,19 @@ def __init__(
             is_valid_file=is_in_split,
         )
 
+    def _load_image(self, index: int) -> Tuple[Tensor, Tensor]:
```

Comment:
```suggestion
    def _load_image(self, index: int) -> tuple[Tensor, Tensor]:
```
We now only support 3.9+